content/bayes_nn/bayes.md

```diff
@@ -0,0 +1,318 @@
+# Bayesian Neural Network
+
+The usual Neural Network are optimized in way to get fixed value of weights and biases that allows the model perform specific task successfully. Instead in 
```

Comment:
"Usually, neural networks are optimized in order to get a fixed value for the weights and biases which allow the model perform a specific task successfully."